shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/common/statement/ddl/DropFunctionStatement.java

```diff
@@ -22,5 +22,5 @@
 /**
  * Drop function statement.
  */
-public final class DropFunctionStatement extends AbstractSQLStatement implements DDLStatement {
+public abstract class DropFunctionStatement extends AbstractSQLStatement {
```

Comment:
I saw none of `xxx DropFunctionStatement` and this class has members, so what role does `DropFunctionStatement` play?

Reply:
> I saw none of `xxx DropFunctionStatement` and this class has members, so what role does `DropFunctionStatement` play?

@tristaZero `DropFunctionStatement` has two implementation classes——`MySQLDropFunctionStatement` and `PostgreSQLDropFunctionStatement`, and they have been included in PR.

![image](https://user-images.githubusercontent.com/10829171/92881428-8c2c8180-f441-11ea-932b-11cd7d177112.png)

![image](https://user-images.githubusercontent.com/10829171/92881464-951d5300-f441-11ea-903c-115af558c5fd.png)

Reply:
Why do all of them have no members?  `MySQLDropFunctionStatement` and `PostgreSQLDropFunctionStatement ` will be filled with member variables later?

Reply:
> Why do all of them have no members? `MySQLDropFunctionStatement` and `PostgreSQLDropFunctionStatement ` will be filled with member variables later?

@tristaZero Yes, they will be optimized in #6498. Later, the table name in the statement will be parsed and the table will be verified.